route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -97,6 +97,7 @@ void setManeuverLaneletIds(carma_planning_msgs::msg::Maneuver& mvr, lanelet::Id
     config_.lateral_accel_limit_ = declare_parameter<double>("vehicle_lateral_accel_limit", config_.lateral_accel_limit_);
     config_.stopping_accel_limit_multiplier_ = declare_parameter<double>("stopping_accel_limit_multiplier", config_.stopping_accel_limit_multiplier_);
     config_.min_maneuver_length_ = declare_parameter<double>("vehicle_id", config_.min_maneuver_length_);
```

Comment:
Looks like this should be config_.vehicle_id
Didn't notice till I saw the added min_maneuver_length below

Reply:
changed